aspnetcore/getting-started.md

```diff
@@ -23,6 +23,8 @@ uid: getting-started
     cd aspnetcoreapp
     dotnet new web
     ```
+    
+    Note: If you get an error running `dotnet new web` or you must run  `dotnet new -t web`, you'll need to upgrade to the current version of .NET Core. Install [.NET Core](https://microsoft.com/net/core)
```

Comment:
They don't absolutely have to upgrade .NET Core. How about something less obtrusive like:

Note: Previous versions of .NET Core required a `t` parameter, that is   `dotnet new -t web`.    `dotnet` (with no parameters)  will display the .NET Core version.